frontend/src/screens/settings/DebugTools.tsx

```diff
@@ -178,6 +179,12 @@ function RefundSwapDialogContent() {
       <AlertDialogHeader>
         <AlertDialogTitle className="capitalize">Refund Swap</AlertDialogTitle>
         <AlertDialogDescription className="flex text-foreground flex-col gap-4">
+          <div className="flex flex-row gap-1 items-center text-muted-foreground">
+            Only On-chain {"->"} Lightning swaps need to be refunded
+            <ExternalLink to="https://guides.getalby.com/user-guide/alby-hub/faq/what-happens-if-lose-access-to-my-hub-while-a-swap-is-in-progress">
```

Comment:
Can we change the link to https://guides.getalby.com/user-guide/alby-hub/faq/what-happens-if-lose-access-to-my-hub-while-a-swap-is-in-progress#swap-out-lightning-on-chain to be more precise?